modules/sdk-core/src/bitgo/lightning/iLightning.ts

```diff
@@ -0,0 +1,164 @@
+import * as t from 'io-ts';
+
+export interface CreateInvoiceParams {
+  amount: number;
+  memo?: string;
+  expiration?: number;
+}
+
+export interface CreateDepositAddressParams {
+  amount: number;
+  memo?: string;
+  expiration?: number;
+}
+
+export interface GetInvoicesParams {
+  id?: string;
+  status?: 'open' | 'closed';
+}
+
+export interface LightningWithdrawalParams {
+  amount: number;
+  address?: string;
+  sequenceId?: string;
+}
+
+export interface LightningDepositParams {
+  amount: number;
+  address?: string;
+  sequenceId?: string;
+}
+
+export interface PayInvoiceParams {
+  invoice: string;
```

Comment:
We do have `memo`, `sequenceId`, `feeLimitRatio`, and `feeLimit` as optional parameters as well when paying a lightning invoice via the API.